tools/homebrew-start/lanraragi

```diff
@@ -1,22 +1,35 @@
 #!/bin/bash
 
-trap "kill 0" SIGINT
-
-# prefix/place where lrr is installed & modify redis config
+# prefix/place where lrr is installed
 workdir=$(dirname $0)/$(dirname $(readlink $0))/..
 
+# initialize LRR_DATA_DIRECTORY if not set
+if [[ ${LRR_DATA_DIRECTORY} == "" ]]; then
+  if [[ ${OSTYPE} == "darwin"* ]]; then
+    _LRR_DATA_DIRECTORY="${HOME}/Library/Application Support/LANraragi"
```

Comment:
If we have a custom LOG_DIR for darwin, it'd be better to move the `export` statement to a dedicated block in order to remove an indentation level here: 

```
if [[ ${OSTYPE} == "darwin"* ]]; then
  _LRR_DATA_DIRECTORY="${HOME}/Library/Application Support/LANraragi"
  _LRR_LOG_DIRECTORY="${HOME}/Logs/LANraragi"
else
  _LRR_DATA_DIRECTORY="${HOME}/LANraragi"
  _LRR_LOG_DIRECTORY="${HOME}/LANraragi/log"
fi

if [[ ${LRR_DATA_DIRECTORY} == "" ]]; then
  export LRR_DATA_DIRECTORY="${_LRR_DATA_DIRECTORY}"
fi

if [[ ${LRR_LOG_DIRECTORY} == "" ]]; then
  export LRR_LOG_DIRECTORY="${_LRR_LOG_DIRECTORY}"
fi
```

Reply:
added!